lib/js/jsfetch.nim

```diff
@@ -0,0 +1,120 @@
+## - Fetch for the JavaScript target: https://developer.mozilla.org/docs/Web/API/Fetch_API
+from httpcore import HttpMethod
+export HttpMethod
+
+when not defined(js) and not defined(nimdoc):
```

Comment:
just `when not defined(js) and not defined(nimdoc):`

now that `nim doc -b:js main` works